packages/Dispatch/readme.md

```diff
@@ -217,6 +237,7 @@ With the following options
 | --luisSubscriptionRegion| (optional) Cognitive Service LUIS region from portal.azure.com  |
 | --dispatch              | (optional) .dispatch file path    |
 | --dataFolder            | (optional) Dispatch working directory |
+| --doAutoActiveLearning  | (optional) Default to false. If true, will also run 'test' against the local model created during the auto active leanring process |
```

Comment:
Misspelling: leanring

(optional) Default to false. If true, will also run 'test' against the local model created during the auto active learning process